### ISSUE TITLE
Add fields to batch update

### ISSUE THREAD

**Issue by danny248** [InQuest/ThreatKB#351]
c2ip/c2dns - expiration_timestamp, category
yara rule - category

**Comment by dcuellar322** [comment]
@danny248 

c2ip/c2dns do not have a Category field in the data model. Did you want it added?

I went ahead and implemented batch edit for Expiration Timestamp for both c2ip/c2dns.

Yara Rule batch edit for Category has already been implemented #347 ... is it not working as expected?

Here's expiration timestamp added to batch edit of c2ips:
![image](https://user-images.githubusercontent.com/3661852/65299187-19d9d780-db34-11e9-8de6-9cc7c2c686dd.png)

Here's expiration timestamp added to batch edit of c2dns:
![image](https://user-images.githubusercontent.com/3661852/65299201-31b15b80-db34-11e9-84b7-aab7d6186b47.png)
